docs/src/osdk/reference/commands/build.md

```diff
@@ -29,6 +29,11 @@ Build artifacts in release mode, with optimizations
 - `--features <FEATURES>`:
 Space or comma separated list of features to activate
 
+- `--no-default-features`
+Do not activate the `default` features
+
+More Cargo options will be supported in future versions of OSDK.
```

Comment:
The `--config` option of Cargo is supported as well. Maybe document it here too.

https://github.com/asterinas/asterinas/blob/e0c6c294815d23800f3437bbb1b755f635dec338/osdk/src/cli.rs#L238-L244